feature/splash/src/main/java/com/ku_stacks/ku_ring/splash/SplashViewModel.kt

```diff
@@ -0,0 +1,40 @@
+package com.ku_stacks.ku_ring.splash
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.ku_stacks.ku_ring.space.repository.KuringSpaceRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class SplashViewModel @Inject constructor(
+    private val kuringSpaceRepository: KuringSpaceRepository
+) : ViewModel() {
+
+    private val _splashScreenState = MutableStateFlow(SplashScreenState.INITIAL)
+    val splashScreenState: StateFlow<SplashScreenState>
+        get() = _splashScreenState
```

Comment:
```suggestion
    val splashScreenState: StateFlow<SplashScreenState> = _splashScreenState.asStateFlow()
```

별반 차이없을 수 있지만 MutableStateFlow를 ReadonlyStateFlow로 변환해준다는 점에서 의미가 있어보이네요.

> ReadOnlyStateFlow 의 계층을 따라가면 emit 을 가지는 Interface를 구현하지 않는다. 따라서 asStateFlow 로 하면 어떤 형변환을 해도 값을 발행 할 수 없다. 외부에서 값 발행을 반드시 제한해야 하는 경우 적절하다

```kotlin
public fun <T> MutableStateFlow<T>.asStateFlow(): StateFlow<T> =
    ReadonlyStateFlow(this, null)
```


[참고](https://medium.com/@equipment_room/kotlin-mutableshared-stateflow-to-shard-stateflow-984f69989d57)

Reply:
큰 차이는 없는 것 같지만, 습관으로 해두기 좋아보이네요. `StateFlow`를 다시 `MutableStateFlow`로 변환하는 발상은 (안티패턴이긴 하지만) 처음 봤네요;;

https://github.com/ku-ring/KU-Ring-Android/pull/151/commits/193950ad2313a2e862b08373851ecdd4d1cd1b16 에서 수정했습니다.